websites/R/Radiko/dist/metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Munn",
+    "id": "379616881272946688"
+  },
+  "contributors": [
+    {
+      "name": "Krauchelli",
+      "id": "229334514084741120"
+    }
+  ],
+  "service": "Radiko",
+  "description": {
+    "en": "Japanese's radio broadcast platform."
+  },
```

Comment:
there is probs also a jp description you can add.

Reply:
Updated